File: Service_eHospital_DongNai_A_NSTL/Implement/NS_RP_DSTableService.cs
namespace Service_eHospital_DongNai_A_NSTL.Implement
{
    public class NS_RP_DSTableService : BaseService<NS_RP_DSTable, INS_RP_DSTableRepository>
    , INS_RP_DSTableService
    {
    private readonly INS_RP_DSTableRepository _NS_RP_DSTableRepository;
    public NS_RP_DSTableService(INS_RP_DSTableRepository NS_RP_DSTableRepository) : base(NS_RP_DSTableRepository)
    {
    _NS_RP_DSTableRepository = NS_RP_DSTableRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A_NSTL/NS_RP_DSTable.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NS_RP_DSTable } from './NS_RP_DSTable.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NS_RP_DSTableService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'Ma', 'Vietnamese', 'SapXep'];

    List: NS_RP_DSTable[] | undefined;
    ListFilter: NS_RP_DSTable[] | undefined;
    FormData!: NS_RP_DSTable;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NS_RP_DSTable";
    }
}

